### PR TITLE
Query Multiple tables

### PR DESCRIPTION
**Pull request by gracekabaghe**
- Use primary key & foreign key mechanisms for joining tables.
- Query multiple tables.
- Prepare complex queries that answer analytical questions.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -43,3 +45,89 @@ VALUES ('Boarmon', '06-07-2005', '7', 'true', '20.4');
...
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') 
+WHERE name = 'Angemon';
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') 
+WHERE name = 'Boarmon';
+
+
```
Comment (ElijahTobs) [inline data.sql]: - [x] `animals` table should be modified such that if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @gracekabaghe 👋

You've done really well so far! 👍 

## Major Highlights 💯 

Your README is professional and ✔️ 
You've followed the Gitflow rules. ✔️ 
You have good commit messages ✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️


_See the inline comments below_


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: data.sql (hunk 1 above)
Comment (gracekabaghe) [reply data.sql]: DONE

File: data.sql [hunk 2]
```diff
@@ -43,3 +45,94 @@ VALUES ('Boarmon', '06-07-2005', '7', 'true', '20.4');
...
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') 
+WHERE name = 'Angemon';
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') 
+WHERE name = 'Boarmon';
+
+BEGIN;
+UPDATE animals SET species = 'Digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'Pokemon' WHERE species IS NULL;
```
Comment (SajjadAhmad14) [inline data.sql]: Please update `species_id`  to `Digimon` if the animal's name ends with `mon` and all other animals should have `Pokemon`.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Grace, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 2 above)
Comment (gracekabaghe) [reply data.sql]: I'm not sure I know what you mean, are you referring to a result like this: https://share.getcloudapp.com/v1uLbQrJ/annotate

**Review by AshakaE — APPROVED** [review]
### Hi @gracekabaghe   🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**